lib/bots/versionbot.ts

```diff
@@ -92,20 +91,26 @@ interface ExternalCommand {
     };
 }
 
+enum ReviewState {
+    Approved = 0,
+    ChangesRequired
+};
```

Comment:
Oh right. No, a comment is considered a 'Well, ok then', just like we're doing with these NITs. :)